src/app/services/logger/exui-logger/exui-logger.component.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { TestBed, async } from '@angular/core/testing';
```

Comment:
this needs to be moved to app/component/exu-logger

Reply:
Yes, why is the component test in the services?

Reply:
Done